src/utils/frontend-api-client/schedule-of-rates/codes.js

```diff
@@ -16,10 +16,15 @@ export const getSorCodes = async (
   return data
 }
 
-export const getSorCode = async (code, propertyReference) => {
+export const getSorCode = async (
+  code,
+  propertyReference,
+  contractorReference = false
+) => {
   const { data } = await axios.get(`/api/schedule-of-rates/codes/${code}`, {
     params: {
       propertyReference: propertyReference,
+      ...(contractorReference && { contractorReference: contractorReference }),
```

Comment:
I think we could skip specifying the property here as a micro-improvement

`...(contractorReference && { contractorReference }),`

Reply:
Ahh nice syntax knowledge here 😉